### DISCUSSION TITLE
Error 500 internal server.

### DISCUSSION THREAD

**Discussion by SylenthMen** [vercel/community#5916] [Help]
### Summary

Hello, I have these errors in the log, this happens when I upload a file to my application

console error

POST https://chatea-con-el-pdf.vercel.app/api/upload 500 (Internal Server Error)
o	@	App.qxyaF65a.js:4
(anónimo)	@	App.qxyaF65a.js:4
(anónimo)	@	App.qxyaF65a.js:4
(anónimo)	@	App.qxyaF65a.js:4
Promise.then (asincrónica)		
be

error 1
[POST] /api/upload status=500

error 2
(node:8) ExperimentalWarning: buffer.File is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

error 3

Unhandled Promise Rejection 	{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"Error: ENOENT: no such file or directory, open '/var/task/public/text/afec3be72c5c9459e0dbf413deb557a7.txt'","reason":{"errorType":"Error","errorMessage":"ENOENT: no such file or directory, open '/var/task/public/text/afec3be72c5c9459e0dbf413deb557a7.txt'","code":"ENOENT","errno":-2,"syscall":"open","path":"/var/task/public/text/afec3be72c5c9459e0dbf413deb557a7.txt","stack":["Error: ENOENT: no such file or directory, open '/var/task/public/text/afec3be72c5c9459e0dbf413deb557a7.txt'"]},"promise":{},"stack":["Runtime.UnhandledPromiseRejection: Error: ENOENT: no such file or directory, open '/var/task/public/text/afec3be72c5c9459e0dbf413deb557a7.txt'","    at process.<anonymous> (file:///var/runtime/index.mjs:1276:17)","    at process.emit (node:events:529:35)","    at emit (node:internal/process/promises:149:20)","    at processPromiseRejections (node:internal/process/promises:283:27)","    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)"]}

error 4

Unknown application error occurred

### Example

https://chatea-con-el-pdf.vercel.app/

### Steps to Reproduce

Select a PDF file from your computer and in Console you will see error 500

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to showing what the function does? That would be helpful for collaborative debugging 🙏
